docs/api.md

```diff
@@ -272,6 +273,39 @@ Failing requests may be due to the following errors:
 * status code 400, errno 107:  request query contains invalid parameters
 * status code 400, errno 108:  request query missing required parameters
 
+## POST /v1/account/status
+
+Gets the status of an account without exposing user data
```

Comment:
docs nit: it's not clear what "exposing user data" means, can we say something more concrete?  It may also be worth noting explicitly that this endpoint is rate-limited, and listing the matching "rate limit exceeded" error code in the section on failing requests.